doc/03-Configuration.md

```diff
@@ -93,3 +105,18 @@ allowing to keep this information for longer with a smaller storage footprint.
 
 A duration string is a sequence of decimal numbers and a unit suffix, such as `"20s"`.
 Valid units are `"ms"`, `"s"`, `"m"` and `"h"`.
+
+### Galera Cluster
+
+Icinga DB expects a more consistent behaviour from its database than a [Galera cluster](https://mariadb.com/kb/en/what-is-mariadb-galera-cluster/) provides by default.
+To accommodate this, Icinga DB sets the [wsrep_sync_wait](https://mariadb.com/kb/en/galera-cluster-system-variables/#wsrep_sync_wait) system variable for all its database connections. Consequently,
```

Comment:
Now the lines are getting really long here. You might want to reflow the sections you added to 120 characters length (that's what most[^1] lines in this file outside of tables adhere to).

[^1]: There's one line with 121 characters. :(